snakedenfitness/fitness/models.py

```diff
@@ -31,6 +31,10 @@ class Workout(models.Model):
     reps = models.IntegerField()
     sets = models.IntegerField()
     weight = models.IntegerField()
+<<<<<<< HEAD
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+=======
```

Comment:
this is a merge conflict that needs fixed

Reply:
fixed